src/coreservices.cpp

```diff
@@ -104,6 +106,21 @@ inline QLocale inputLocale() {
 
 namespace mixxx {
 
+// Function to check if a given file is a playlist
+bool isPlaylist(QString fileName) {
+    // Create a list of known playlist file extensions
+    QStringList playlistExtensions;
+    playlistExtensions << "m3u"
+                       << "m3u8"
+                       << "pls";
```

Comment:
This will initialize the list in a single run. 

```suggestion
    const QStringList playlistExtensions = {
            QStringLiteral("m3u"),
            QStringLiteral("m3u8"),
            QStringLiteral("pls")};
```
Not sure if we need QString(QStringLiteral("m3u")) ..